.make/dependencies/rmUnnecessaryFolders.php

```diff
@@ -33,13 +31,34 @@
 
 clearstatcache();
 
-$oiTopComposer = new iTopComposer();
-$aDeniedButStillPresent = $oiTopComposer->ListDeniedButStillPresent();
+// Read params
+$key = array_search("--manager", $argv);
+if (false === $key || false === isset($argv[$key + 1]) ) {
+	throw new \Exception("Usage: " . __FILE__ . " --manager composer|npm");
```

Comment:
Use one of the typed SPL Exception ? 
Ref https://www.php.net/manual/en/spl.exceptions.php
I would choose InvalidArgumentException ?

```suggestion
	throw new \InvalidArgumentException("Usage: " . __FILE__ . " --manager composer|npm");
```